### DISCUSSION TITLE
About Intro Texts

### DISCUSSION THREAD

**Discussion by ChillyBeanBAM** [FunkinCrew/Funkin#3964] [Q&A] [answered]
Is there a way to add more intro texts that are seen before the title screen without replacing the pre-existing ones? What I'm saying is that I want to put them in a seperate introText.txt file in my mod folder.

**Comment by MAZ12211** [comment, marked as answer]
[You can append your intro texts at the end of the file](https://funkincrew.github.io/funkin-modding-docs/10-appending-and-merging-files/10-01-appending-files.html#appending-to-txt-files)

**Reply by ChillyBeanBAM** [reply]
Oh, thank you! I did not see that section! 😅
